src/smc-webapp/frame-editors/latex-editor/pdfjs.tsx

```diff
@@ -464,7 +467,34 @@ class PDFJS extends Component<PDFJSProps, PDFJSState> {
     return <div style={{ background: "yellow" }} id={"cc-highlight"} />;
   }
 
+  render_no_pdf(): Rendered {
+    return (
+      <div style={{ backgroundColor: "white" }}>
+        {" "}
+        <p>There is no rendered PDF file available.</p>
+        {this.props.derived_file_types.size > 0 ? (
+          <p>
+            Instead, you might want to switch to the{" "}
+            {list_alternatives(this.props.derived_file_types)} view by selecting
+            it via the dropdown selector in the button row above.
+          </p>
+        ) : (
+          ""
+        )}
+      </div>
+    );
+  }
+
   render() {
+    if (
+      this.props.mode == "rmd" &&
+      this.props.derived_file_types != undefined
+    ) {
+      if (!this.props.derived_file_types.contains("pdf")) {
```

Comment:
like in the html iframe case, rmd mode and we know about alternatives but there is no pdf → give user a hint about what's going on